moveit_ros/robot_interaction/src/robot_interaction.cpp

```diff
@@ -91,10 +91,9 @@ void RobotInteraction::decideActiveComponents(const std::string& group, Interact
 {
   decideActiveEndEffectors(group, style);
   decideActiveJoints(group);
-  if (active_eef_.empty() && active_vj_.empty() && active_generic_.empty())
+  if (!group.empty() && active_eef_.empty() && active_vj_.empty() && active_generic_.empty())
     ROS_INFO_NAMED("robot_interaction", "No active joints or end effectors found for group '%s'. "
-                                        "Make sure you have defined an end effector in your SRDF file and that "
-                                        "kinematics.yaml is loaded in this node's namespace.",
+                                        "Make sure that kinematics.yaml is loaded in this node's namespace.",
```

Comment:
Is this necessary? This code runs in RViz right?
`kinematics.yaml` is only loaded into the `move_group` namespace, isn't it?

Reply:
No, rviz has it's a copy of kinematics.yaml loaded into its own (private) namespace. rviz doesn't access the move_group namespace. However, we might want to change _this_. There is no reason to have an independent set of parameters for moveit, indeed.
I would propose to make a new PR for this.

Reply:
It *what*? I am not aware of that.
If you start an anonymous rviz instance from the command line, you can still use the MotionPlanningDisplay.
Is there a fallback in there already to look in `move_group` too?

Either way, the rviz plugin should get the parameters from the move_group namespace.

Reply:
> If you start an anonymous rviz instance from the command line, you can still use the MotionPlanningDisplay.
> Is there a fallback in there already to look in move_group too?

Interesting question. Can you please open another issue to discuss this...